provide manual instructions in addition to cargo-release

i get a gnarly error cargo-installing cargo-release, to be on the safe side i think including manual docs would be a good call

would be cool if cargo-dist helped solve this error for cargo-release ;) (it looks like an openssl issue but it is too late for me to debug, here's a cut and paste)

```
error: linking with `cc` failed: exit status: 1
  |
  = note: "cc" "-arch" "arm64" "/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/rustcwDPMBh/symbols.o" "/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/cargo_release-bbe8459a41113cd0.cargo_release.301733a3-cgu.0.rcgu.o" "/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/cargo_release-bbe8459a41113cd0.cargo_release.301733a3-cgu.1.rcgu.o" "/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/cargo_release-bbe8459a41113cd0.cargo_release.301733a3-cgu.10.rcgu.o" "/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/cargo_release-bbe8459a41113cd0.cargo_release.301733a3-cgu.11.rcgu.o" "/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/cargo_release-bbe8459a41113cd0.cargo_release.301733a3-cgu.12.rcgu.o" "/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/cargo_release-bbe8459a41113cd0.cargo_release.301733a3-cgu.13.rcgu.o" "/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/cargo_release-bbe8459a41113cd0.cargo_release.301733a3-cgu.14.rcgu.o" "/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/cargo_release-bbe8459a41113cd0.cargo_release.301733a3-cgu.15.rcgu.o" "/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/cargo_release-bbe8459a41113cd0.cargo_release.301733a3-cgu.2.rcgu.o" "/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/cargo_release-bbe8459a41113cd0.cargo_release.301733a3-cgu.3.rcgu.o" "/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/cargo_release-bbe8459a41113cd0.cargo_release.301733a3-cgu.4.rcgu.o" "/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/cargo_release-bbe8459a41113cd0.cargo_release.301733a3-cgu.5.rcgu.o" "/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/cargo_release-bbe8459a41113cd0.cargo_release.301733a3-cgu.6.rcgu.o" "/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/cargo_release-bbe8459a41113cd0.cargo_release.301733a3-cgu.7.rcgu.o" "/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/cargo_release-bbe8459a41113cd0.cargo_release.301733a3-cgu.8.rcgu.o" "/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/cargo_release-bbe8459a41113cd0.cargo_release.301733a3-cgu.9.rcgu.o" "/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/cargo_release-bbe8459a41113cd0.sfg8trqhgfjzben.rcgu.o" "-L" "/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps" "-L" "/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/build/libgit2-sys-593d4cd1192798c1/out/build" "-L" "/usr/local/opt/openssl@1.1/lib" "-L" "/Users/ag_dubs/.rustup/toolchains/stable-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libenv_logger-a5db36dd935e097d.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libhumantime-9f7ce6e4af9851da.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libcargo_release-55f248d8f85b7b1b.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libgit_conventional-949bcbf04bfab227.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libunicase-dc5c91f2d32cffed.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libnom-42ba14482a592cf4.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libclap_cargo-4680185d905d7a41.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libdoc_comment-fd9a928d9edc57a9.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libconcolor_control-737479b8da14b0cb.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libatty-d16a6d1135c599b0.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libconcolor_query-46cde94bd973560e.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libdunce-ed7fbc6b26552971.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libdifflib-2d168ee424951f33.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libcrates_index-00bfdc83bda14bdf.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libhex-411d65781afd70a7.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libhome-0fe5e96110154890.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libtoml-bd86c4e7cdaf0f02.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/librustc_hash-7512336624fa851a.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/librayon-ebd577202e125386.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/librayon_core-e6fb312e52bf92e3.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libnum_cpus-4b9320519244eb26.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libcrossbeam_deque-d11928861821433b.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libcrossbeam_epoch-f1cfb2917fdceef7.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libmemoffset-270af2aaacb7a58f.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libscopeguard-dc6a5ef55a793d41.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libcrossbeam_channel-686a7fe1e5d80edb.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libcrossbeam_utils-234b2d84e0b2133d.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libsmartstring-2551b32fb0523aed.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libstatic_assertions-1f1ff57ecff92569.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libgit2-cfed5c2d953990be.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/liburl-aed1c6111046073b.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libidna-994fb0fb5f6af643.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libunicode_normalization-85438f4387f38447.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libtinyvec-499ac370b8d3c4f1.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libtinyvec_macros-c16760f6c0fb0217.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libunicode_bidi-cf5624381bf37bda.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libform_urlencoded-056e0562607d49c7.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libpercent_encoding-ce59dd7370ff8540.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/liblibgit2_sys-820f2ca43fc9cfc3.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libopenssl_sys-40a51cccebecc746.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/liblibz_sys-a52c305c65b1c6f8.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libignore-a91750060b945d90.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/liblazy_static-a079c1087d5e6dba.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libwalkdir-392059ad31eaa52f.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libsame_file-7c7ad9662f50202a.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libthread_local-ea6d0db24ad8c253.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libglobset-0de34d74e7e56f2a.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libfnv-617038571a2f9d4d.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libregex-bdbbb26d85d9eef8.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libregex_syntax-289134a5cfa8367f.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libaho_corasick-8dc878e60e7cc5d4.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libdirs_next-da2191f7e010ef11.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libdirs_sys_next-37e67c65b96848f9.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libtoml_edit-4a91af565901576f.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libindexmap-0b29ad2ece4d7131.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libhashbrown-57f6b81eaec80abe.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libtoml_datetime-12288df4a799cf3c.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libnom8-7142d92ab1ec5095.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libcargo_metadata-852b3279303e3471.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libserde_json-01cdc5af4792f459.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libryu-7af3dcfe64a22b02.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libthiserror-9bff314f9671bbad.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libcargo_platform-1d683307f404897c.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libsemver-c45f705e95195b01.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libcamino-03a3e6a88a6328c5.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libmaplit-a0cd4e39e1ac20ac.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libtime-f63e389d749c893f.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libitoa-8b0a713281ea956d.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libtime_core-219574701200be86.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/liblog-fd0a5259dcd4cb44.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libcfg_if-7ed28c9907d719bf.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libitertools-4469ed5d1108498e.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libeither-6c69fd690d358154.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libanyhow-1cc22f00d070939b.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libbstr-0fdd62be75120ac7.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libregex_automata-b4df3e5d0b9e47f6.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libmemchr-bcc8e18b5a52df9e.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libserde-f5057923ac835ec7.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libclap-11e86b1fae87971c.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libterminal_size-c22bcbc76f5136e8.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libstrsim-fc6901605e5a07e6.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libis_terminal-02528b9061a56d5a.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/librustix-5400c090b52913f5.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/liberrno-4359f6e641f437e2.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libio_lifetimes-2257b487fc7223a9.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/liblibc-ca4d8f60f1026952.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libclap_lex-d049f1fbe64a107e.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libos_str_bytes-880c6acd8524a7cc.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libtermcolor-9fede4ad5af9114b.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libbitflags-abddf8c59d4d8b5b.rlib" "/private/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/libonce_cell-0370a6c62bfa46ff.rlib" "/Users/ag_dubs/.rustup/toolchains/stable-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib/libstd-8b48f6cefff7b1fb.rlib" "/Users/ag_dubs/.rustup/toolchains/stable-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib/libpanic_unwind-77aeaad8f850e496.rlib" "/Users/ag_dubs/.rustup/toolchains/stable-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib/libobject-4374cc9e76b48de5.rlib" "/Users/ag_dubs/.rustup/toolchains/stable-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib/libmemchr-71ce82846aaede9b.rlib" "/Users/ag_dubs/.rustup/toolchains/stable-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib/libaddr2line-88478a43231eec41.rlib" "/Users/ag_dubs/.rustup/toolchains/stable-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib/libgimli-219a0c42f67a2534.rlib" "/Users/ag_dubs/.rustup/toolchains/stable-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib/librustc_demangle-967edde4cff44b15.rlib" "/Users/ag_dubs/.rustup/toolchains/stable-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib/libstd_detect-5e9bad2c7ec1f53a.rlib" "/Users/ag_dubs/.rustup/toolchains/stable-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib/libcfg_if-f2d3de7828f2e1f2.rlib" "/Users/ag_dubs/.rustup/toolchains/stable-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib/libhashbrown-39feb0179b2f197f.rlib" "/Users/ag_dubs/.rustup/toolchains/stable-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib/libminiz_oxide-fe7523d51d4e2154.rlib" "/Users/ag_dubs/.rustup/toolchains/stable-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib/libadler-1fb439e2b08a54ae.rlib" "/Users/ag_dubs/.rustup/toolchains/stable-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib/librustc_std_workspace_alloc-0ece1873dc3050cd.rlib" "/Users/ag_dubs/.rustup/toolchains/stable-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib/libunwind-1094a874bdfc7dbb.rlib" "/Users/ag_dubs/.rustup/toolchains/stable-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib/libcfg_if-8cd76113b92d6775.rlib" "/Users/ag_dubs/.rustup/toolchains/stable-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib/liblibc-3a7cb21921429c0e.rlib" "/Users/ag_dubs/.rustup/toolchains/stable-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib/liballoc-5b42fb3902d391c7.rlib" "/Users/ag_dubs/.rustup/toolchains/stable-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib/librustc_std_workspace_core-b495777441600026.rlib" "/Users/ag_dubs/.rustup/toolchains/stable-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib/libcore-cba95c4386a21469.rlib" "/Users/ag_dubs/.rustup/toolchains/stable-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib/libcompiler_builtins-b8f163a13cf94b5d.rlib" "-liconv" "-framework" "Security" "-framework" "CoreFoundation" "-lssl" "-lcrypto" "-lz" "-liconv" "-lSystem" "-lc" "-lm" "-L" "/Users/ag_dubs/.rustup/toolchains/stable-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib" "-o" "/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38/release/deps/cargo_release-bbe8459a41113cd0" "-Wl,-dead_strip" "-nodefaultlibs"
  = note: ld: warning: ignoring file /usr/local/opt/openssl@1.1/lib/libssl.dylib, building for macOS-arm64 but attempting to link with file built for macOS-x86_64
          ld: warning: ignoring file /usr/local/opt/openssl@1.1/lib/libcrypto.dylib, building for macOS-arm64 but attempting to link with file built for macOS-x86_64
          Undefined symbols for architecture arm64:
            "_OPENSSL_init_ssl", referenced from:
                std::sync::once::Once::call_once::_$u7b$$u7b$closure$u7d$$u7d$::hee03dd9df5577a1d in libopenssl_sys-40a51cccebecc746.rlib(openssl_sys-40a51cccebecc746.openssl_sys.85336bb9-cgu.4.rcgu.o)
                core::ops::function::FnOnce::call_once$u7b$$u7b$vtable.shim$u7d$$u7d$::h91bade0e91e44e64 in libopenssl_sys-40a51cccebecc746.rlib(openssl_sys-40a51cccebecc746.openssl_sys.85336bb9-cgu.4.rcgu.o)
          ld: symbol(s) not found for architecture arm64
          clang: error: linker command failed with exit code 1 (use -v to see invocation)


error: could not compile `cargo-release` due to previous error
error: failed to compile `cargo-release v0.24.4`, intermediate artifacts can be found at `/var/folders/7g/wqzllfcj31zgt2k60nxb1dwc0000gn/T/cargo-installkfjW38`
```

--- a/README.md
+++ b/README.md
@@ -56,6 +56,25 @@ cargo release 0.1.0
 
 That's gonna do a whole bunch of stuff you might not have expected (but I left off the --execute flag from `cargo-release` so you won't actually break anything if you really did just copy paste that 😇). Read the rest of the docs to know what it does!
 
+Or if you don't want to use cargo-release, you can emulate its behaviour manually:
+
+```sh
+# install tools
+cargo install cargo-dist
+
+# one-time setup
+cargo dist init --ci=github
+git add .
+git commit -am "wow shiny new cargo-dist CI!"
+
+# cut a release
+# <first manually update the version of your crate, run tests, etc>
+git commit -am "chore: Release version 0.1.0"
+git tag v0.1.0
+cargo publish
+git push
+git push --tags
+```
 
 
 
@@ -189,9 +208,18 @@ cargo-release will then automatically:
 4. Publish to crates.io (disable this with `--no-publish`)
 4. Push to your repo's main branch
 
-You don't actually need to use cargo-release, it's just a great tool for streamlining your release process that we recommend everyone using!
+You don't actually need to use cargo-release, it's just a great tool for streamlining your release process that we recommend everyone using! Here's roughly a manual version of what it does in simple cases:
 
-When you *do* push a tag the CI will take over and do the following:
+```sh
+# <first manually update the version of your crate, run tests, etc>
+git commit -am "chore: Release version 0.1.0"
+git tag v0.1.0
+cargo publish
+git push
+git push --tags
+```
+
+When you *do* push a tag (and the commit it points to) the CI will take over and do the following:
 
 1. Create a *draft* Github Release™️ (with [taiki-e/create-github-release-action](https://github.com/taiki-e/create-gh-release-action))
 2. Build your application for all the target platforms, wrap them in zips/tars, and upload them to the Github Release™️

--- a/README.md
+++ b/README.md
@@ -40,6 +40,31 @@ That said, the current version is Very Very Unstable And Experimental and the ex
 
 # Way-Too-Quick Start
 
+
+```sh
+# install tools
+cargo install cargo-dist
+
+# one-time setup
+cargo dist init --ci=github
+git add .
+git commit -am "wow shiny new cargo-dist CI!"
+
+# cut a release like you normally would
+
+# <manually update the version of your crate, run tests, etc>
+# then:
+git commit -am "chore: Release version 0.1.0"
+git tag v0.1.0
+cargo publish
+git push
+git push --tags
+```
+
+That's gonna do a whole bunch of stuff you might not have expected, but if it all works you'll get a Github Release™️ with built and zipped artifacts uploaded to it! Read the rest of the docs to learn more!
+
+You may have noticed "cut a release" still has a lot of tedious work. That's because we recommend using [cargo-release](https://github.com/crate-ci/cargo-release) to streamline the last step:
+
 ```sh
 # install tools
 cargo install cargo-dist
@@ -54,27 +79,7 @@ git commit -am "wow shiny new cargo-dist CI!"
 cargo release 0.1.0
 ```
 
-That's gonna do a whole bunch of stuff you might not have expected (but I left off the --execute flag from `cargo-release` so you won't actually break anything if you really did just copy paste that 😇). Read the rest of the docs to know what it does!
-
-Or if you don't want to use cargo-release, you can emulate its behaviour manually:
-
-```sh
-# install tools
-cargo install cargo-dist
-
-# one-time setup
-cargo dist init --ci=github
-git add .
-git commit -am "wow shiny new cargo-dist CI!"
-
-# cut a release
-# <first manually update the version of your crate, run tests, etc>
-git commit -am "chore: Release version 0.1.0"
-git tag v0.1.0
-cargo publish
-git push
-git push --tags
-```
+(I left off the --execute flag from `cargo-release` so you won't actually break anything if you really did just copy paste that 😇)
 
 
 
@@ -187,10 +192,21 @@ Other commands like `cargo dist build` (bare `cargo dist`) will always default t
 
 Once you've completed setup (run `cargo dist init --ci=...`), you're ready to start cutting releases!
 
-The github workflow will trigger whenever you push a [git tag](https://git-scm.com/book/en/v2/Git-Basics-Tagging)
-to the main branch of your repository that looks like a version number (`v1`, `v1.2.0`, `v0.1.0-prerelease2`, etc.).
+The github workflow will trigger whenever you push a [git tag](https://git-scm.com/book/en/v2/Git-Basics-Tagging) to the main branch of your repository that looks like a version number (`v1`, `v1.2.0`, `v0.1.0-prerelease2`, etc.).
 
-The easiest way to do this is to use [cargo-release](https://github.com/crate-ci/cargo-release):
+You might do that with something like this:
+
+```sh
+# <first manually update the version of your crate, run tests, etc>
+# then:
+git commit -am "chore: Release version 0.1.0"
+git tag v0.1.0
+cargo publish
+git push
+git push --tags
+```
+
+That's a bunch of junk to remember to do, so we recommend using [cargo-release](https://github.com/crate-ci/cargo-release) to streamline all of that:
 
 ```sh
 cargo release 0.1.0
@@ -207,17 +223,6 @@ cargo-release will then automatically:
 3. Make a git tag
 4. Publish to crates.io (disable this with `--no-publish`)
 4. Push to your repo's main branch
-
-You don't actually need to use cargo-release, it's just a great tool for streamlining your release process that we recommend everyone using! Here's roughly a manual version of what it does in simple cases:
-
-```sh
-# <first manually update the version of your crate, run tests, etc>
-git commit -am "chore: Release version 0.1.0"
-git tag v0.1.0
-cargo publish
-git push
-git push --tags
-```
 
 When you *do* push a tag (and the commit it points to) the CI will take over and do the following:
 

